### ISSUE TITLE
Offline Online subscribe event Not invoking in Safari Browser Mac

### ISSUE THREAD

**Issue by riyasksalim** [cyrilletuzi/ngx-pwa-offline#6]
It seems the offline-online subscribe event is not working on the Safari Browser Versions such as 11.1.2  and 12. But it is working on the Safari browser Version 11.1.1 and all other browsers

Reference url : 
https://stackblitz.com/edit/ngx-pwa-offline-demo
![offline_issue](https://user-images.githubusercontent.com/15179953/51530933-0de12d00-1e62-11e9-9ecd-ca6bbd5ce404.png)

**Comment by cyrilletuzi** [comment]
There was already an issue before the `online`/`offline` event is not always attached to the same object in all browsers (sometimes it's `navigator`, or `window`...). So it may have changed in Safari.

Could you investigate?

**Comment by riyasksalim** [comment]
yes it tried the navigator.online in the respective safari browser and it is working .seems like you code is using the window . am not for sure but this is my findings please do try to fix
![code](https://user-images.githubusercontent.com/15179953/51581260-542f9e00-1eed-11e9-9d30-1036d4eed250.JPG)

**Comment by cyrilletuzi** [comment]
Your stackblitz demo works on Safari 12.0.2. I've also done my own reproduction and it works.

What are you doing to go "offline"?

Be aware that the native `online` events and property are not always reliable. For example, if your connected to a network (Wifi or LAN) but with no access to Internet, the navigator will still says it's online.
